cmd/snap/main.go

```diff
@@ -36,6 +37,14 @@ var optionsData options
 
 var parser = flags.NewParser(&optionsData, flags.HelpFlag|flags.PassDoubleDash)
 
+// ClientConfig is the configuration of the client.Client used by all commands.
```

Comment:
just "for the Client" instead "of the client.Client", we see the full type below anyway